### DISCUSSION TITLE
如何设置多签地址为owner地址，及如何从owner 地址为多签地址重新设置成正常的地址？

### DISCUSSION THREAD

**Discussion by Reskorey** [filecoin-project/community-china#117] [Bug讨论]
有没有哪个兄弟测试过Lotus v1.11.0 的功能，把owner 地址设置成多签地址？我这边通过这个方法试了：https://github.com/filecoin-project/lotus/pull/5342 ，并且可以从miner 地址中提取挖矿收益，但是现在发现一个问题是没办法把多签的owner 地址设置成正常的 f1 或者f3 地址？求教

**Comment by dcshy** [comment]
自有技术团队近百人， 机房密封日产能2p+ ， 独立节点集群方案， 可加vx：IPFS-Shy
